src/lib/retire-js/RetireJsEngine.ts

```diff
@@ -318,9 +320,11 @@ export class RetireJsEngine extends AbstractRuleEngine {
 		// Theoretically, the output is at least a valid JSON.
 		let outputJson = null;
 		try {
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
It's fine to assign this `any` here, because we validate that it's actually a `RetireJsOutput` a few lines later.